lib/bridge/github/issues.rb

```diff
@@ -16,9 +16,9 @@ def issues(label = nil, opts={})
 
       issues_response = gh.issues(params) do |request|
         request.headers["Accept"] = "application/vnd.github.squirrel-girl-preview"
-      end
+      end.all
 
-      issues_response.all.each{
+      issues_response.each{
```

Comment:
Is thay a quirk of Ghee's proxy implementation, or something inherent to Ruby itself? Assigning an intermediate variable should never implicitly change behavior. :frowning: